packages/google-analytics/README.md

```diff
@@ -1,10 +1,67 @@
-> ## Beta version
-> Please, bear in mind that this package is currently in beta version. It hasn't proper tests or docs and its implementation may change with the final release.
```

Comment:
We still need to maintain a warning about the beta version.

Reply:
Done! 👍